### DISCUSSION TITLE
Question about the navego application

### DISCUSSION THREAD

**Discussion by Adrienmangini** [rodralez/NaveGo#130] [Q&A] [answered]
Hello to all, I would like, from my IMU, to bring out the data of angles, distance and speeds, without taking into account the GPS.
So it seems to me that the functions are in the inss file. 
Which csv format should I load and how to apply the functions on this please file. 
My file is the following:

accelerometer x, y z
gyroscope x,y,z 
magnetometer x,y,z

Thank you very much, nice work.
[acc_Course_Moi.csv](https://github.com/rodralez/NaveGo/files/8992295/acc_Course_Moi.csv)

**Comment by rodralez** [comment, marked as answer]
Hello Adrien,


You can use the function `sensor_read()` to read a csv file. Then, you have to transform this data to NaveGo's input format. 

Next, some example code:

```
clc
close all
clear
matlabrc

addpath /path_to/NaveGo/data-acquisition/

sensor_data = sensor_read('/path_to_file/acc_Course_Moi.csv', 1, 1, ';');

imu.t = sensor_data.Timestamp;
imu.wb = [sensor_data.X sensor_data.Y sensor_data.Z ];
```
